platforms/core-execution/persistent-cache/src/main/java/org/gradle/cache/CacheBuilder.java

```diff
@@ -60,9 +39,9 @@ enum LockTarget {
     /**
      * Specifies the <em>initial</em> lock options to use. See {@link PersistentCache} for details.
      *
-     * Note that not every combination of cache type and lock options is supported.
+     * @param mode the lock mode to use
      */
-    CacheBuilder withLockOptions(LockOptions lockOptions);
```

Comment:
@big-guy is `CacheBuilder` internal API?

Otherwise, I think this is a breaking change in 8.7 RC1 because of the removed `CacheBuilder.withLockOptions` method.